public/js/graph/react_graph.js

```diff
@@ -1194,8 +1199,7 @@ var Bool = React.createClass({
 
 var EdgeGroup = React.createClass({
     // EdgeGroup's state is used to keep track of the edgeIDs of
-    // edges that are missing. Void is just a placeholder state so
-    // we can declare an initial state; it does nothing.
+    // edges that are missing.
```

Comment:
This particular change should be done in a separate pull request to `master`, not here. It doesn't relate to the Draw tool.